tests/forms_tests/field_tests/test_choicefield.py

```diff
@@ -52,6 +52,12 @@ def test_choicefield_4(self):
         with self.assertRaisesMessage(ValidationError, msg):
             f.clean('6')
 
+    def test_choicefield_choices_default(self):
+        f = ChoiceField()
+        # The default choices kwarg tuple is coerced to list in the property
```

Comment:
Maybe it's not important to mention this implementation detail which could change?